lib/backend/k8s/k8s.go

```diff
@@ -296,6 +296,9 @@ func (c *KubeClient) Clean() error {
 		apiv3.KindGlobalNetworkSet,
 		apiv3.KindIPPool,
 		apiv3.KindHostEndpoint,
+		apiv3.KindBlockAffinity,
```

Comment:
Hm, did this change work? Do the tests pass? I thought i had trouble when doing this, but it was a loong time ago.